client/src/components/container/admin/LogInPage/index.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import { Button, Form, Input, Alert, message, Spin } from 'antd';
+import { MailOutlined, LockOutlined } from '@ant-design/icons';
+
+import './style.css'
+import Logo from './images/logo.svg';
+import adminLoginImg from './images/adminlogin.png';
+
+class AdminLogin extends Component {
+    state = {
+        error: '',
+        isLoaded: false,
+    }
+
+    onFinish = ({ email, password }) => {
+        this.setState({ isLoaded: true })
+        axios.post(`/api/v1/login`, { email, password }).then(({ data }) => {
+            const { history: { push } } = this.props;
+            message.success(data.msg)
+            push('/admin/dashboard')
+        }).catch(({ response: { data: { msg } } }) => {
```

Comment:
Personal opinion but I think 2 - level destructuring is really hard to read. It'll also throw an error if the response is something that you don't expect.

What happens here for example if the user's internet connection drops temporarily? I think you'll get an error but not in the format that you're expecting. In these situations I would check to see if you can get `response.data.message` but if it's not available provide a fallback message